styles.css

```diff
@@ -413,6 +413,104 @@ svg {
   color: #fff;
 }
 
+.contact_form {
+  display: flex;
+  flex-flow: column;
+}
+
+.contact_title {
+  margin: 69px 5% 60px 5%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: 90%;
+  height: 216px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 90%;
+  margin-left: 5%;
+}
+
+form input,
+form textarea {
+  width: 100%;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  border: none;
+  margin-bottom: 10px;
+}
+
+form textarea {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  font-style: italic;
+  height: 114px;
+  background-color: #fbf8f7;
+}
+
+form ul {
+  list-style: none;
+  width: 100%;
+}
+
+form ul li {
+  border-bottom: 1px solid #dbd8d7;
+  margin: 27px 0;
+  width: 100%;
+}
+
+.contact_button {
+  margin-top: 24px;
+  margin-bottom: 78px;
+  padding: 12px;
+  border: none;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  background: #ff6b00;
+}
+
+.contact_button:hover {
+  padding: 11px;
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  cursor: pointer;
+  color: #f55800;
+}
+
+footer ul {
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  margin-top: 14px;
+  padding-bottom: 54px;
+}
+
+footer ul li {
+  margin: 11px;
+}
+
+footer ul li img {
+  width: 20px;
+  height: 20px;
+}
+
```

Comment:
- [ ] According to the project requirements we must stick to the [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=1yfXx6yoiCqxC0tL-0) and make sure our design looks identical to figma. So please make sure your design also includes all the elements present in the figma design as your design is missing a line under the footer icons

Figma design

<img width="435" alt="Screenshot 2023-03-28 at 10 12 01 PM" src="https://user-images.githubusercontent.com/44798044/228319131-edbbb126-35b2-484d-ac01-563270b0c6d3.png">


Your design:


<img width="417" alt="Screenshot 2023-03-28 at 10 23 08 PM" src="https://user-images.githubusercontent.com/44798044/228319261-2de4415d-fac2-4df8-bd1b-01f9292e6650.png">